src/main/java/amazonreviewsapp/springboot/controller/ReviewController.java

```diff
@@ -0,0 +1,116 @@
+package amazonreviewsapp.springboot.controller;
+
+import amazonreviewsapp.springboot.dto.EditReviewRequestDto;
+import amazonreviewsapp.springboot.dto.ReviewRequestDto;
+import amazonreviewsapp.springboot.dto.ReviewResponseDto;
+//import amazonreviewsapp.springboot.jwt.JwtTokenProvider;
+import amazonreviewsapp.springboot.mapper.ReviewMapper;
+import amazonreviewsapp.springboot.model.Review;
+import amazonreviewsapp.springboot.model.User;
+import amazonreviewsapp.springboot.service.ReviewService;
+import amazonreviewsapp.springboot.service.UserService;
+import java.security.InvalidKeyException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/reviews")
+public class ReviewController {
+    @Autowired
+    private ReviewService reviewService;
+
+    @Autowired
+    private ReviewMapper mapper;
+
+    @Autowired
+    private UserService userService;
+
+   /* @Autowired
+    private JwtTokenProvider provider;*/
+
+    @GetMapping
+    @RequestMapping("/most-used-words")
+    public List<Object> findMostUsedWords(@RequestParam String limit) {
+        Object[] words = reviewService.findMostUsedWordsFromReviews();
+        return Arrays.stream(words)
+                .limit(Long.parseLong(limit))
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping
+    @RequestMapping("/add-review")
```

Comment:
```suggestion
    @RequestMapping("/")
```
по тому, що це пост зрозуміло, що ми хочемо додати ресурс